angular/projects/admin-nrpti/src/app/mines/mines-list/mines-list.component.html

```diff
@@ -1,4 +1,40 @@
 <main class="container-fluid-padding">
+  <h1>All Mines</h1>
+  <br />
+  <div class="border-bottom">
+    <div class="row mb-3">
+      <div class="col">
+        <span style="font-size: 1.25rem;">Keyword Search</span>
```

Comment:
Can we remove the inline CSS?

Reply:
Yep. I just copied from the search in records, but I can change it in both places and move it into the CSS file.